schema/codelists/nameType.csv

```diff
@@ -1,8 +1,6 @@
-code,title,description,technical note
-individual,Given name
-translation,Translation of name
-former,Former name
-alias,Alias
-aka,Also Known As
-nick,Nickname
-birth,Birth name
\ No newline at end of file
+code,title,description,technical note,
+individual,Legal name,"A name that identifies the person for legal, administrative and other official purposes. A person's legal name will usually be the one that is found on official government documents.",,
+translation,Translation of name,"A translation of the person's individual name in a different language.",,
```

Comment:
I have a Q in the issue about whether we want transliterated as an option here.

Reply:
Edited as per discussion in #179